LICENCE

```diff
@@ -22,6 +22,8 @@ Copyright (c) 2017-2021
   Microsoft, for contributions by Robert Norton-Wright and Nathaniel Wesley Filardo
   Peter Rugg
   Aril Computer Corp., for contributions by Scott Johnson
+  Philipp Tomsich
+  VRULL GmbH, for contributions by its employees
```

Comment:
Are you covered by the latter or not? Also don't know if "its employees" is sufficient here, so far everyone has been explicitly named

Reply:
Technically not, as control contravenes employment in our geography.
We could make it "employees and contractors", though...